torch/lib/c10d/reducer.cpp

```diff
@@ -988,6 +988,12 @@ void Reducer::prepare_for_forward() {
   std::lock_guard<std::mutex> lock(mutex_);
   num_iterations_++;
   forward_start_time_ = current_time_in_nanos();
+  // Log runtime (e.g. avg performance) stats at randomly selected iterations.
```

Comment:
"10, 1000, or 10000" are not randomly selected iterations and can cause confusion. Could you please update the comment for the consistency?

Reply:
sounds good!